File: Python/11.automorphicNumber.py
def autoNumber(n):
    s = pow(n,2)
    m = pow(10, len(str(n)))
    if (s % m == n):
        print("It's an Automorphic Number")
    else:
        print("It's not an Automorphic Number")
        
autoNumber(376)
autoNumber(377)

File: JS/number/11.automorphicNumber.js
function autoNumber(n) {
  var s = Math.pow(n, 2);
  var m = Math.pow(10, n.toString().length);
  if (s % m == n) {
    console.log("It's an Automorphic Number");
  } else {
    console.log("It's not an Automorphic Number");
  }
}

autoNumber(376);
autoNumber(377);
